### PR TITLE
Mentioning that VAPID is for scaffolding API specs, not developing complete specs

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/design-create-publish-api-specs.adoc [hunk 1]
```diff
@@ -7,6 +7,6 @@ In API Designer, you can use two different tools to create API specifications in
...
 
-Visual API editor:: Use this tool if you do not have much experience with writing API specifications. As you fill in various forms, the editor writes your API specification for you. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual API Editor in API Designer].
+Visual API editor:: Use this tool for lightweight designing and creating scaffolds. This editor does not allow you to import RAML fragments, add security schemes, or perform several of the tasks that the API editor allows. As you fill in various forms, the editor writes your API specification for you. After creating a scaffold in this editor, you can switch to the API editor to customize your specification. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual API Editor in API Designer].
```
Comment (mnbattaglia) [inline modules/ROOT/pages/design-create-publish-api-specs.adoc]: I would remove the "add security schemes" limitation as, in fact, we will be adding this in the next version

File: modules/ROOT/pages/design-create-publish-api-visual-editor.adoc [hunk 2]
```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
...
 
 == Before you begin
 Within Anypoint Platform, your user ID must be assigned the _Design Center Developer_ permission.
 
 == About this task
 
-As you populate the forms, the visual API editor generates two API specifications: one in RAML 1.0, the other in OAS 2.0 (in JSON). Both are visible in a read-only pane on the right of the screen. If at any time you want to edit the specification directly, click *Edit RAML*  or *Edit OAS*to switch to the API editor. After you switch, however, you cannot switch back to the visual API editor to continue creating the API specification.
+As you populate the forms, the visual API editor generates two API specifications: one in RAML 1.0, the other in OAS 2.0 (in JSON). Both are visible in a read-only pane on the right of the screen. If at any time you want to edit the specification directly or customize it (import RAML fragments, add a security scheme, and more), click *Edit RAML*  or *Edit OAS* to switch to the API editor. After you switch, however, you cannot switch back to the visual API editor to continue creating the API specification.
```
Comment (mnbattaglia) [inline modules/ROOT/pages/design-create-publish-api-visual-editor.adoc]: Again, I would remove the "add security schemes" limitation as, in fact, we will be adding this in the next version

File: modules/ROOT/pages/design-create-publish-api-specs.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/design-create-publish-api-specs.adoc]: Done.

File: modules/ROOT/pages/design-create-publish-api-visual-editor.adoc (hunk 2 above)
Comment (rddheath) [reply modules/ROOT/pages/design-create-publish-api-visual-editor.adoc]: Done.

File: modules/ROOT/pages/design-create-publish-api-specs.adoc [hunk 3]
```diff
@@ -7,6 +7,6 @@ In API Designer, you can use two different tools to create API specifications in
...
 
-Visual API editor:: Use this tool if you do not have much experience with writing API specifications. As you fill in various forms, the editor writes your API specification for you. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual API Editor in API Designer].
+Visual API editor:: Use this tool for lightweight designing and creating scaffolds. This editor does not allow you to import RAML fragments or perform several of the tasks that the API editor allows. As you fill in various forms, the editor writes your API specification for you. After creating a scaffold in this editor, you can switch to the API editor to customize your specification. The steps for using this tool are described in xref:design-create-publish-api-visual-editor.adoc[Create and Publish an API Specification with the Visual API Editor in API Designer].
```
Comment (mcherem) [inline modules/ROOT/pages/design-create-publish-api-specs.adoc]: Nice rewrite!

**Pull request merged**